### ISSUE TITLE
Portals not working in 1.2.3

### ISSUE THREAD

**Issue by wilkemeyer** [rwtema/extrautilities_old_issues#553]
I've updated from 1.2.2,

built the new Portal to The Last Millenium, a right-click does nothing.

Even the existing - older - Portal to The Deep Dark does'nt work anymore.

**Comment by Carbon-Key** [comment]
Same here, I'm fine if there is a fancy ritual we have to do now. I just want to know what it is :)

**Comment by darkdata** [comment]
same issue here to

**Comment by Chromotron** [comment]
Can also confirm, but it seems to work for some. Reverting to 1.2.2 fixes it.

**Comment by Carbon-Key** [comment]
But if I revert to 1.2.2 then I don't get to go to the cool new millennium! :P

**Comment by spocane** [comment]
Can confirm this issue still present with 1.2.4

**Comment by Carbon-Key** [comment]
Yup, here as well. I deleted the config and regenerated it on the off chance there was some new line in it I didn't see which was disabling the portals.  No such luck. :(

**Comment by Chromotron** [comment]
Did some testing: this problem happens if and only if I have GalactiCraft installed.

**Comment by Carbon-Key** [comment]
Oh, very interesting information.  I too have galacticraft in my pack. It is also very strange though because the default dimension ID's for the moon etc. are -26 through -28, which is nowhere near conflicting with -100 and -112.

**Comment by duck1123** [comment]
I am also having this problem since 1.2.3. Also, since 1.2.4, waila is showing the portal to the last millennium as entity.GalacticraftCore.CelestialScreen.name

https://dl.dropboxusercontent.com/u/270347/2015-04-07_22.17.27.png

**Comment by devryb** [comment]
Same 1.2.3

**Comment by lambaline** [comment]
It actually looks like you have to go change a number in the GalaticCraft config, there's something in there called the "Entity ID for (hidden) Celestial Selection Entity, must be unique." I went ahead and changed this from 184 to a larger number. 

EDIT: No luck. :(

**Comment by draeath** [comment]
So, this may be related...

I've noticed that the player class is changed when you have galacticraft installed. Here's an example, from my server log:
[23:32:16] [Server thread/INFO] [Waila]: Player GCEntityPlayerMP['Panzer'/1530065, l='Ardonia', x=-11525.55, y=102.00, z=-9267.26] connected. Sending ping

Note "GCEntityPlayerMP"

**Comment by draeath** [comment]
This is fixed by installing Player API as -instructed- on the Galacticraft compatibility wiki page. Links:
http://wiki.micdoodle8.com/wiki/Compatibility#Any_other_mod_with_.27Fake_Player.27_detection
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1277996

**Comment by monkeybee11** [comment]
added player API the portal to the last millenium still dosent work :(

**Comment by grydian** [comment]
Make sure you extract the PlayerAPI-1.7.10-1.3.jar from the zip file that is downloaded and add it to the mods folder.  The zip file holds several ways of installing the mod.  Since this is for extra utilities the best way is the forge mod version.
